system/default_ad_api_helpers/ad_api_adaptors/README.md

```diff
@@ -24,6 +24,7 @@ The clear API is called automatically before setting the route.
 | ------------ | ------------------ | ------------------------------------- | -------------------------------------------------- |
 | Subscription | ~/input/fixed_goal | /planning/mission_planning/goal       | The goal pose of route. Disable goal modification. |
 | Subscription | ~/input/rough_goal | /rviz/routing/rough_goal              | The goal pose of route. Enable goal modification.  |
+| Subscription | ~/input/reroute    | /rviz/routing/reroute                 | The goal pose of reroute.                          |
```

Comment:
```suggestion
| Subscription | ~/input/reroute    | /aou/routing/reroute                 | The goal pose of reroute.                          |
```
Wouldn't it be better to change the rviz side instead of changing the ad api?

Reply:
This topic is not an API, so I didn't use the name of the API. This is a topic only between rviz and the adapter for rviz, the adaptor calls the API when rviz publish this topic.